HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,48 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is the platform specific path separator.
+
+method::isVowel
+
+returns:: true if the char is a vowel.
+
+method::isControl
+
+returns:: true if the char is a control character.
```

Comment:
I suggest placing these methods in the order they are defined in https://github.com/supercollider/supercollider/blob/develop/SCClassLibrary/Common/Core/Char.sc

Reply:
ok. thank